.opspec/release/to-docker-cli/op.yml

```diff
@@ -0,0 +1,55 @@
+name: release/to-docker-cli
```

Comment:
@lmarszal how do you feel about adding a check in the entrypoint.sh of the existing Dockerfile which looks for `/var/run/docker.sock` and if it exists it doesn't start a docker daemon and instead uses it? Everything else is same right?

Reply:
basically make this part optional based on existence of docker socket mounted at conventional path https://github.com/opctl/opctl/blob/master/.opspec/release/to-docker/entrypoint.sh#L3-L23

Reply:
I did thought about that. My thinking was to make it look like `docker` image on docker hub.
Docker has two sets of images:
 * `docker:latest` - that has only cli
 * `docker:dind` - that has dockerd and requires privileged mode
I was thinking about the same pattern here (`opctl` having didn and `opctl:cli` having only cli).
But if we don't see benefits in shipping opctl image with cli only your solution sounds better.

Reply:
either seems reasonable. I was just debating a little on additional maintenance of 2 images but no major concerns. Pick your poison ; )